website/source/docs/auth/aws-ec2.html.md

```diff
@@ -1131,9 +1142,11 @@ instance can be allowed to gain in a worst-case scenario.
 <dl class="api">
   <dt>Description</dt>
   <dd>
-   Fetch a token. This endpoint verifies the pkcs#7 signature of the instance identity document.
-   Verifies that the instance is actually in a running state. Cross checks the constraints defined
-   on the role with which the login is being performed.
+Fetch a token. This endpoint verifies the pkcs7 signature of the instance
```

Comment:
You can set wrapping such that it'll keep the indentation of the first line. I'm not sure exactly what, but I can try to parse that out of my vimrc if you like.